### ISSUE TITLE
Supplier invoice ODT template

### ISSUE THREAD

**Issue by ems-co** [Dolibarr/dolibarr#31520]
### Bug

When creating an ODT template for the supplier invoice and adding the tag {object_already_payed} to the document, the returned value is 0 and it does not take into account the payments that has been done to the invoice! 

and the {object_remain_to_pay} is showing the total value of the invoice, even though there are some payments done!

### Dolibarr Version

19.0.3

### Environment PHP

8.3

### Environment Database

Mysql

### Steps to reproduce the behavior and expected behavior

1- Create an ODT template for Vendor invoice and add these tags {object_already_payed} {object_remain_to_pay}
2- create an invoice for a vendor and make some payments 
3- there will be the ODT document generated but the {object already payed} tag will show zero and the remain to pay will show the total amount even though there are some payments

**Comment by MC874** [comment]
This is true. Still happening on the latest version of Dolibarr. @ems-co request to re-open.

<img width="664" height="166" alt="Image" src="https://github.com/user-attachments/assets/40ae47e9-9ca6-4719-a110-cfa33e10adb1" />
